includes/definitions.inc.php

```diff
@@ -1363,7 +1363,7 @@
 $config['os'][$os]['icon'] = 'hikvision';
 
 // Canopy / Cambium support
-$os = 'canopy';
```

Comment:
Canopy is only labeled for the FSK and 450 equipment. When it came to the ptp 650/250 or the epmp equipment they wouldn't be considered canopy so it seemed more appropriate to generalize them all as cambium. The same detection that was being used for those canopy is still in place.